packages/app-utils/src/plugins/PluginUtils.tsx

```diff
@@ -147,28 +117,45 @@ export function getAuthHandlers(
 export function getAuthPluginComponent(
   pluginMap: PluginModuleMap,
   authConfigValues: Map<string, string>,
-  corePlugins?: Map<string, AuthPlugin>
+  corePlugins = new Map<string, AuthPlugin | LegacyAuthPlugin>()
 ): AuthPluginComponent {
   const authHandlers = getAuthHandlers(authConfigValues);
   // Filter out all the plugins that are auth plugins, and then map them to [pluginName, AuthPlugin] pairs
   // Uses some pretty disgusting casting, because TypeScript wants to treat it as an (string | AuthPlugin)[] array instead
-  const authPlugins = (
-    [...pluginMap.entries()].filter(
-      ([, plugin]: [string, { AuthPlugin?: AuthPlugin }]) =>
-        isAuthPlugin(plugin.AuthPlugin)
-    ) as [string, { AuthPlugin: AuthPlugin }][]
-  ).map(([name, plugin]) => [name, plugin.AuthPlugin]) as [
-    string,
-    AuthPlugin,
-  ][];
+  const legacyAuthPlugins: AuthPlugin[] = (
+    [...pluginMap.entries()].filter(([, plugin]) => 'Component' in plugin) as [
```

Comment:
This is incorrect. See my comment on `LegacyAuthPlugin` below, but you're filtering out the `auth-keycloak` plugin.